addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -573,6 +586,58 @@ export class TablePlugin extends Plugin {
         }
     }
 
+    onKeyDownArrows(ev) {
+        const selection = this.shared.getSelectionData().deepEditableSelection;
+        const anchorNode = selection.anchorNode;
+        const currentCell = closestElement(anchorNode, "td");
+        const currentTable = closestElement(anchorNode, "table");
+        if (!selection.isCollapsed || !currentCell) {
+            return;
+        }
+        const isArrowUp = ev.key === "ArrowUp";
+        const cellPosition = { row: getRowIndex(currentCell), col: getColumnIndex(currentCell) };
+        const tableRows = [...currentTable.rows].map((row) => [...row.cells]);
+        const shouldNavigateCell = (currentNode) => {
+            let node = currentNode;
+            const siblingDirection = isArrowUp ? "previousElementSibling" : "nextElementSibling";
+            while (node && node !== currentCell) {
+                // If a sibling is found, verify that it is not an inline element.
+                if (
+                    node[siblingDirection] &&
+                    (node[siblingDirection].nodeType !== Node.ELEMENT_NODE ||
+                        isBlock(node[siblingDirection]))
```

Comment:
This would break for `<td>abc<br>d[]ef</td>` because `<br>` is not a block element so I would suggest.
```suggestion
                    node[siblingDirection] &&
                    (isBlock(node[siblingDirection]) || node[siblingDirection].nodeName === "BR")
```